tests/ui/type/type-check/point-at-inference-3.stderr

```diff
@@ -1,8 +1,10 @@
 error[E0308]: mismatched types
-  --> $DIR/point-at-inference-3.rs:7:12
+  --> $DIR/point-at-inference-3.rs:8:12
    |
 LL |     v.push(0i32);
-   |            ---- this is of type `i32`, which causes `v` to be inferred as `Vec<i32>`
+   |     -      ---- this argument has type `i32`...
+   |     |
+   |     ... which constrains `v` to have type `Vec<i32>`
```

Comment:
This presentation I guess could use some tweaking. Also the wording "which constrains" feels a bit... mathematical. Open to tweaks.

Reply:
“forces”? idk

Reply:
"forces"/"makes"/"causes"?